editor/plugins/org.fusesource.ide.project/src/org/fusesource/ide/project/camel/CamelArtifactAdapter.java

```diff
@@ -0,0 +1,49 @@
+/*******************************************************************************
+ * Copyright (c) 2007 - 2013 Red Hat, Inc.
```

Comment:
Is it a moved file that github wasn't able to find? or a new one? in this the copyright date is not correct